src/posix/platform/radio_spinel.cpp

```diff
@@ -1073,6 +1112,18 @@ void RadioSpinel::RadioTransmit(void)
                     mTransmitFrame->mInfo.mTxInfo.mMaxCsmaBackoffs, mTransmitFrame->mInfo.mTxInfo.mMaxFrameRetries,
                     mTransmitFrame->mInfo.mTxInfo.mCsmaCaEnabled);
 
+    if (mTransmitFrame->mInfo.mTxInfo.mCsmaCaEnabled)
+    {
+        mTxRadioEndUs = otSysGetTime() + MIN_TX_WAIT_US +
+                        mTransmitFrame->mInfo.mTxInfo.mMaxFrameRetries *
+                            (1UL << (MAC_MIN_BE + mTransmitFrame->mInfo.mTxInfo.mMaxFrameRetries +
+                                     mTransmitFrame->mInfo.mTxInfo.mMaxCsmaBackoffs));
```

Comment:
Say `mMaxFrameRetries = 16` (which is what we use typically), and the `mMaxCsmaBackoffs` is 32 (default OT value). If I am not mistaken the equation  would then give us  `16 * (1UL << (3 + 16 + 32)) = 16 * (1UL << 51)  = 2^4*2^51=2^55` in usec which is very huge number (in order of years I think). 

I suggest  we use a reasonable long wait time (~5 seconds) independent of the CSMA attempts or number of retries. 

Thoughts?

Reply:
Updated accordingly

Reply:
I think the equation was not correct. I might be:
```python
UNIT_BACKOFF_PERIOD * mMaxFrameRetries * ((mMaxCsmaBackoffs - macMaxBe + 1) * 2^macMaxBe - 1)
```
However, we may enhance later.